html/semantics/interfaces.html

```diff
@@ -27,7 +33,9 @@
 }
 
 elements.forEach(function(a) {
-  do_test(a[0], a[1]);
-  do_test(a[0].toUpperCase(), a[1]);
+  do_test(a[0], a[1], true);
```

Comment:
1) Boolean parameters are unintelligible when reading the caller.  What does "true" mean here?  Even if you look it up once, it's hard to remember after that.  Change this to a string or something, please.

2) Why are you using the the NS variant for one and not the other?  Just for slightly more completeness without testing everything twice?